### ISSUE TITLE
The blog URL is not matching the current blog URLs

### ISSUE THREAD

**Issue by kennethlarsen** [ember-learn/empress-blog-ember-template#19]
Current: https://ember-ghost-ember-template.netlify.com/the-editor

Expected: https://ember-ghost-ember-template.netlify.com/YEAR/MONTH/DAY/the-editor

**Comment by mansona** [comment]
so this isn't something that we are going to be able to fix from the template perspective 🤔 what the current plan is that we are going to create netlify redirects in https://github.com/ember-learn/ember-blog/pull/7 that will cover the URL update

does that make sense?

**Comment by kennethlarsen** [comment]
Right, makes sense. Just made a brain dump of what I saw missing 😅
